src/pages/kb/open-source/admin-guide/secrets.md

```diff
@@ -0,0 +1,63 @@
+---
+category: admin-guide
+parent_category: open-source
+title: Secrets Keys in Redash
+slug: secrets
+toc: true
+---
+
+## Background
+Redash encrypts secret information with two keys: the Cookie Secret and the Data Source Secret.
+
+### Cookie Secret 
+
+The cookie secret is taken from the `REDASH_COOKIE_SECRET` environment variable. It is used for various cryptographic features of the web server, such as authenticating users, signing cookies, and storing user session information. It is required to start the application.
+
+If you attempt to start Redash without it, the webserver will not initialise and you will find the following message in your logs:
+
+```bash
+Exception: You must set the REDASH_COOKIE_SECRET environment variable. Visit http://redash.io/help/open-source/admin-guide/secrets for more information.
+```
+
+To fix this message: create an environment variable called `REDASH_COOKIE_SECRET` and give it a value.
+### Data Source Secret
+
+The data source secret is taken from the `REDASH_SECRET_KEY` environment variable. It is used to encrypt database passwords, API keys, connection strings, and any other secret fields found on the Settings > Data Sources screen.
```

Comment:
We encrypt the whole configuration object of the data source / alert destination - not only the secrets.

Reply:
Fixed.